### ISSUE TITLE
iOS Simulator freezes after loading WDA and app

### ISSUE THREAD

**Issue by PraveenKumarQA** [appium/appium#11863]
## The problem
Appium reloads existing ios simulator and it freezes. It doesn't show app or wda loading but it runs test fine. Simulator screen stays frozen even after test is over, I have to restart the simulator to make simulator functional again.

## Environment
Last working configuration: (App testing was working and simulator was not freezing and showing the test happening)
Appium : 1.8
Node: 8
Xcode: 9.4.1
OS - 11.1 (iOS)
Phone Name - iPhone 7 (Simulator)

Issue first noticed: ( App testing was working but simulator freezes after simulator is started via Appium, it didn't show loading of WDA or App)
Appium : 1.10
Node: 10
Xcode: 10.1
OS - 11.1 (iOS)
Phone Name - iPhone 7 (Simulator)

Currently still seeing issue: ( App testing was working but simulator freezes after simulator is started via Appium, it didn't show loading of WDA or App)
Appium : 1.7.2
Node: 7.9.0
Xcode: 9.2
OS - 11.1 (iOS)
Phone Name - iPhone 7 (Simulator)

## Details
Everything was working great until I tried to update my xcode to 10.1 and appium 1.10. Once I updated and ran my test, it would relaunch the simulator but it wouldn't show WDA and App being installed and simulator would be frozen. Test would run at the background but doesn't show anything on simulator. Once test is complete simulator is still frozen. I have to restart the simulator to see the WDA and App was installed. I downgraded to earlier configuration i.e Xcode-9.4.1 and appium - 1.8 and still facing same issue. I further downgrade to Xcode -9.2 and Appium -1.7.2 and still facing same issue (i.e that test would run but it doesn't show on simulator. Simulator screen is frozen and to see if app and wda was installed, we need to restart the simulator). Any guidance would be really helpful.

## Link to Appium logs
<script src="https://gist.github.com/PraveenKumarQA/7d31cc000b9a5e24a1442ab0d038175c.js"></script>

## Code To Reproduce Issue [ Good To Have ]
import io.appium.java_client.AppiumDriver;
import io.appium.java_client.TouchAction;
import io.appium.java_client.android.AndroidDriver;
import io.appium.java_client.ios.IOSDriver;
import io.appium.java_client.remote.MobileCapabilityType;
import org.openqa.selenium.By;
import org.openqa.selenium.Dimension;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.interactions.touch.TouchActions;
import org.openqa.selenium.remote.DesiredCapabilities;

import java.net.MalformedURLException;
import java.net.URL;
import java.util.Set;

import static utility.webDrivers.DriverAppium.iosport;

/**
 * Created by praveenkumar on 6/16/18.
 */
public class AppiumSample {
    public static void main(String[] args) throws InterruptedException, MalformedURLException {

        AppiumDriver<WebElement> appiumDriver;

        DesiredCapabilities driverCap = new DesiredCapabilities();
        // Simulator Capabilities
        driverCap.setCapability("deviceName", "iPhone 7");
        driverCap.setCapability("platformName", "iOS");
        driverCap.setCapability("platformVersion", "11.1");
        driverCap.setCapability("app", "/Users/praveenkumar/Downloads/UICatalog.app");
        driverCap.setCapability("locationServicesAuthorized", true);
        driverCap.setCapability("autoAcceptAlerts", true);
        driverCap.setCapability("locationServicesAuthorized", true);
        driverCap.setCapability("setWebContentsDebuggingEnabled", true);
        driverCap.setCapability("automationName", "XCUITest");
        driverCap.setCapability("showXcodeLog", true);
        driverCap.setCapability("showIosLog", true);
        appiumDriver = new IOSDriver<>(new URL("http://127.0.0.1:4723" + "/wd/hub"), driverCap);

    }
}

**Comment by mykola-mokhnach** [comment]
Duplicate of https://github.com/appium/appium/issues/11621

**Comment by PraveenKumarQA** [comment]
The work around mentioned in #11621 worked for me, so what I have done 
* Downloaded and installed Xcode 10.0
* Copied and pasted CoreSimulator.Framework as mentioned in #11621 
* Updated Pom.xml, Selenium-java - 3.9.1, java-client - 6.0.0-BETA5

And it started working, Thanks guys for helping!!!

**Comment by dkolechintsev** [comment]
Thanks a lot, **PraveenKumarQA** 
It works for me
>     * Copied and pasted CoreSimulator.Framework as mentioned in #11621
